internal/plugin/connectors/http/conjur/plugin.go

```diff
@@ -16,16 +18,23 @@ func PluginInfo() map[string]string {
 	}
 }
 
-// NewConnector returns an http.Connector that decorates each incoming http
-// request with a basic auth header.
-func NewConnector(conRes connector.Resources) http.Connector {
-	return &Connector{
-		logger: conRes.Logger(),
-	}
-}
-
 // GetHTTPPlugin is required as part of the Secretless plugin spec for HTTP
 // connector plugins. It returns the HTTP plugin.
 func GetHTTPPlugin() http.Plugin {
-	return http.NewConnectorFunc(NewConnector)
+	newConnector, err := generichttp.NewConnectorConstructor(
+		&generichttp.ConfigYAML{
+			Headers: map[string]string{
+				"Authorization": "Token token={{ `%s` .token | base64 }}",
```

Comment:
Since the token value needs to be surrounded by quotation marks I'd suggest the following:
```
"Authorization": `Token token="{{ .token | base64 }}"`,
```